### DISCUSSION TITLE
Exploring Personal Finance App Development with Expo, Expo Router, and PocketBase

### DISCUSSION THREAD

**Discussion by brunowilliang** [pocketbase/pocketbase#2896] [Show and tell]
Hello fellow developers! I'm excited to share my latest personal project - a finance app I developed using Expo, Expo Router, and the powerful backend capabilities of PocketBase. This project was an exhilarating journey into app development, and I'm thrilled to invite you all to check it out.

The app allows users to manage their personal finances, tracking both income and expenses across multiple categories. It's been an incredible learning experience for me, and I'm eager to get your insights, feedback, and ideas for future improvements. Let's continue to learn and innovate together!

You can find the project at https://github.com/Brunowilliang/my-finance. Looking forward to hearing from you!"

https://twitter.com/Brunowgarcia/status/1678926554917662721

**Comment by shmlkv** [comment]
Great work, @Brunowilliang! Do you have plans to publish backend of this app?

**Reply by brunowilliang** [reply]
Hi @shmlkv 
I created a template on pockethost:
https://myfinance.pockethost.io/_/
email: demo@myfinance.com
password: myfinance@demo

**Comment by kingdevnl** [comment]
Hey @Brunowilliang How did you get arround the Flipper SSE issue? I haven't been able to figure out how to get arround it myself

**Reply by ganigeorgiev** [reply]
@kingdevnl Please check the suggestions from https://github.com/pocketbase/pocketbase/discussions/2484#discussioncomment-6118523.

**Reply by jaspersmitNL** [reply]
@ganigeorgiev That suggestion only works for plain react native not using expo as it needs to update the native app code

**Reply by ganigeorgiev** [reply]
@jaspersmitNL I'm not sure that I understand what do you mean. The above linked Flipper issue doesn't seem reproducible when using expo (see https://github.com/pocketbase/pocketbase/discussions/2484#discussioncomment-5930450), only when using react native on android.

In any case, if you are not able to resolve your issue, please open a new discussion with more details about your setup and how you are testing (_ideally with a **minimal** reproducible example_).
